docs/concepts.rst

```diff
@@ -16,9 +16,14 @@ The Nonprofit Success Pack and the Product Delivery Model
 
 The Nonprofit Success Pack, or NPSP, is one of Salesforce.org's flagship products. The NPSP consists of six managed packages, with complex dependency relationships between them, alongside unpackaged metadata and setup automation that helps create a usable NPSP org.
 
-Salesforce.org delivers NPSP using CumulusCI and the Product Delivery Model because those tools are what make it possible for us to ship this large, complex, heterogeneous application efficiently to tens of thousands of customers — without a lengthy setup guide to be completed by the end user. CumulusCI automation seamlessly deploys all six managed packages in the right sequence, delivers the unpackaged metadata, like Record Types and Page Layouts, we supply to make the customer's life easier, and executes setup automation like populating NPSP relationship values. 
+Salesforce.org delivers NPSP using CumulusCI and the Product Delivery Model because those tools are what make it possible for us to ship this large, complex, heterogeneous application efficiently to tens of thousands of customers — without a lengthy setup guide to be completed by the end user. CumulusCI automation seamlessly
 
-CumulusCI runs this automation throughout our development lifecycle, starting from feature branches in the hands of our developers and culminating the delivery of new versions of the application to our users, including through MetaDeploy — which runs the very same automation we use internally to set up and configure a customer org.
+* deploys NPSP's five dependent managed packages in the right sequence;
+* delivers unpackaged Record Types for the Account and Opportunity objects;
+* installs NPSP;
+* and performs final configuration to make the customer's experience better, like setting up Global Actions and delivering translations.
+
+CumulusCI runs this automation throughout our development lifecycle, starting from feature branches in the hands of our developers and culminating in the delivery of new versions of the application to our users through MetaDeploy.
 
 Throughout the CumulusCI documentation, we'll have the Product Delivery Model in mind. 
 
```

Comment:
```suggestion
**Packages**

If you’ve come this far in your search for automation, we’re guessing you’re familiar with packages as well as their role in customizing a Salesforce org. However, because most everything that is built in CumulusCI is deployed via packages, it’s essential to clarify once more:

A **package** is a container for something as small as an individual component or as large as a set of related apps. After creating a package, you can distribute it to other Salesforce users and organizations, including those outside your company.

**Unmanaged packages** are typically used to distribute open-source (non-proprietary) features or application templates to provide developers with the basic building blocks for an application. Once the components are installed from an unmanaged package into a specific org, it’s what’s known as an org implementation. These freshly installed components can be edited as seen fit by the owners of this implementation. The developer who created and uploaded the unmanaged package has no control over the installed components, and can't change or upgrade them.
 
**Managed packages** are typically used by Salesforce partners to distribute and sell applications to customers. They are proprietary code (apps?) that can only be upgraded and deployed by the developer that built them. To ensure seamless upgrades, certain destructive changes, like removing objects or fields, cannot be performed.

Packages are also built, stored and deployed via a **project**.
```